projects/angular/src/forms/datepicker/_variables.datepicker.scss

```
@@ -25,10 +26,10 @@ $clr-calendar-action-buttons-height: calc(
   button-variables.$clr-btn-height + tokens.$cds-global-space-7
 ); // Button Height + Padding set on .datepicker-actions class
 
-//(8 = 6 date rows + 1 row for day heading + 1 row for calendar switches and month/year pickers) + 2 * 16px padding + 2px for border
+//(8 = 6 date rows + 1 row for day heading + 1 row for calendar switches and month/year pickers) + 2 * 16px padding + 2px for border + 6 * 4px vertical gap for buttons
 $clr-calendar-height: calc(
   calc(8 * #{tokens.$cds-global-space-11}) + calc(2 * #{tokens.$cds-global-space-7}) +
-    calc(2 * #{tokens.$cds-alias-object-border-width-100})
+    calc(2 * #{tokens.$cds-alias-object-border-width-100}) + calc(6 * #{tokens.$cds-global-space-3})
```

Use `$clr-calendar-button-gap` variable.

```suggestion
    calc(2 * #{tokens.$cds-alias-object-border-width-100}) + calc(6 * $clr-calendar-button-gap)
```

---

Fixed in [948b80d](https://github.com/vmware-clarity/ng-clarity/pull/1706/commits/948b80d3d202cba561298fdb5a6636cfd8e84a57)